src/de/quinscape/Gluecksrad.java

```diff
@@ -24,37 +25,32 @@ public void startGame() {
     }
 
     public void randomWordGenerator() {
-        currentWord = wordList().get(randomGenerator.nextInt(wordList().size()));
-    }
-
-    public boolean checkInputLetterAmount(String userInput){
-        if(userInput.length() != 1){
-            return false;
-        }
-        return true;
+        player.setCurrentWord(wordList().get(randomGenerator.nextInt(wordList().size())));
```

Comment:
So eine Konstruktion darf gerne in mehreren Zeilen geschrieben werden. So bleibt es dennoch übersichtlich: 
```suggestion
        int randomIndex = randomGenerator.nextInt(wordList().size());
        player.setCurrentWord(wordList().get(randomIndex));
```